src/parser/elements_parser.ts

```diff
@@ -1,8 +1,9 @@
 //@ts-nocheck
-(function(global) {
-  var fabric = global.fabric;
 
-  fabric.ElementsParser = function(elements, callback, options, reviver, parsingOptions, doc) {
+import { fabric } from "../../HEADER";
+import { capitalize, invertTransform, multiplyTransformMatrices, qrDecompose } from "../util";
```

Comment:
please import them from respective files if migrated. I want to remove that generic util barrel import.